src/Symfony/Component/HttpKernel/DataCollector/ConfigDataCollector.php

```diff
@@ -288,4 +285,108 @@ private function determineSymfonyState()
 
         return $versionState;
     }
+
+    /**
+     * Gets the names, paths and installed versions of the bundles enabled in
+     * the application.
+     *
+     * @return array
+     */
+    private function getBundleData()
+    {
+        $bundles = array();
+        $installedPackages = $this->getInstalledPackages();
+
+        foreach ($this->kernel->getBundles() as $name => $bundle) {
+            $bundlePath = $bundle->getPath();
+            $bundleVersion = null;
+
+            foreach ($installedPackages as $packageName => $packageVersion) {
+                if (strpos($bundlePath.'/', '/vendor/'.$packageName.'/')) {
```

Comment:
You should tes that `strpos` is diffferent than fqlse (`false !== strpos()`. Otherwise if it's 0, it will not detect it